doc/authentication-backends/JWT-authentication-module.md

```diff
@@ -0,0 +1,36 @@
+## Overview
+
+JWT authentication backend can verify JSON Web Tokens provided by the clients.
+A wide range of signature algorithms is supported, including those using public key cryptography.
+
+The module checks the signature and also the validity of the following parameters:
```

Comment:
The module checks signature and validity of the following parameters:

Reply:
The module checks the signature and validity of the following parameters: